packages/client3/services/user-services.ts

```diff
@@ -9,3 +13,15 @@ export const hasUid = (user?: User) => {
       user?.isUsAccreditedIndividual
   );
 };
+
+export const checkWalletAddress = async (
+  wallet?: string,
+  isState1Selected?: boolean
+): Promise<IAccount | undefined> => {
+  if (wallet) {
+    const response = await axios.get(`${baseUrl}/accounts`, {
```

Comment:
I don't think you need to use baseUrl as just using '\' will point to the local domain. If not needed the file under API folder can also be removed
```suggestion
    const response = await axios.get(`/api/accounts`, {
```